edison/README.md

```diff
@@ -31,48 +53,22 @@ Feature extractors can be combined with each via operators to build newer ones.
 For example, a valid operators to conjoin feature extractors with each other to 
 produce a new `FeatureExtractor`. Use the `FeatureUtilities.conjoin` to do this.
 
-## Defining Feature Extractors 
-You can build feature extractors in two ways:
-
-   1. You can manually build a feature extractor by implementing the FeatureExtractor interface.
-   2. You can use a pre-defined
-
-## List of pre-defined features
+### Output features to a file
+You can output the set of `Feature`s returned by any `FeatureExtractor` to an [SVMLite-format](http://svmlight.joachims.org) 
+file in order to be used by any learning software that supports it using the method 
+`WriteSVMLightFormat.writeFeatureExample`.
 
-This section lists the set of pre-defined feature extractors along
+## List of pre-defined feature extractors
+To see the set of pre-defined feature extractors along
 with their description and the `FeatureExtractor` that implements
-them.
+them please visit our [Edison search interface](http://bilbo.cs.illinois.edu:5900). 
 
 ### Bias feature
 The keyword `bias` in a `.fex` specification includes a feature
 that will always be present. This is useful to add a bias feature
 for binary classification.
 
-### Word features
-The following list of feature extractors are operate on the last
-word of the input constituent. They are all defined as static
-members of the class `WordFeatureExtractorFactory`.
-  
```

Comment:
Some of them were there but I've added the rest (in `WordFeatureExtractorFactory`)